core/src/test/java/edu/wpi/grip/util/GRIPCoreTestModule.java

```diff
@@ -69,6 +74,12 @@ public void tearDown() {
     protected void configure() {
         assert setUp : "The GRIPCoreTestModule handler was not set up. Call 'setUp' before passing the injector";
         super.configure();
+        // HTTP server injection bindings
+        bind(GripServer.HttpServerFactory.class).to(GripServerTest.TestServerFactory.class);
+        bind(GripServer.class).asEagerSingleton();
+        bind(MapNetworkPublisherFactory.class)
+                .annotatedWith(Names.named("httpManager"))
+                .to(HttpManager.class);
```

Comment:
Right, but every other test doesn't need to have the HTTPServer bound.

What I'm trying to get at is that there are other tests that don't need the HTTPServer so when dependency injection is used in other tests I don't want another instance of the HTTPServer floating around. Thats why I moved NetworkTables and ROS into a seperate DI module.
I'm trying to avoid having network tables from being instantiated when I don't want it to be.
If there was a problem with network tables it was causing a ripple effect to other tests.